telegram/inlinequeryresultmpeg4gif.py

```diff
@@ -60,6 +62,7 @@ def __init__(self,
                  thumb_url,
                  mpeg4_width=None,
                  mpeg4_height=None,
+                 mpeg4_duration=None,
```

Comment:
We had discussions about this in the past. Should new parameters be added in the location where Telegram documentation added them or in the end of the parameters list (before kwargs) to avoid backward compatibility issues.
I prefer the latter.
@jh0ker what do you think?